policy/release/base_image_registries.rego

```diff
@@ -33,12 +34,13 @@ deny contains result if {
 }
 
 # METADATA
-# title: Base images must be provided
+# title: Base image information was provided
 # description: >-
 #   The attestation must provide the expected information about which base images
-#   were used during the build process.
+#   were used during the build process. The base image information is expected to
+#   be found in a task result called `BASE_IMAGES_DIGESTS`.
```

Comment:
(not to be addressed in this PR, but commenting as I see it) Are task results like these configurable to make it easier to modify the policies for various applications as others might adopt it?

Reply:
Yes and no. Yes in the sense that you can write your own custom rego to use whatever field you want. No, in the sense that you can't easily use this rule with a different task result name.

Reply:
If we did want to make it more customizable, the var
```
build_base_images_digests_result_name := "BASE_IMAGES_DIGESTS"
```
could switched to rule data, e.g.
```
build_base_images_digests_result_name := lib.rule_data("base_image_digests_result_name")
```
with a suitable default value.

Then the user would have a nice way to use this same rule with their own preferred task result name.